commune_app/all_models/votes.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from .chores import Chore
+from .users import User
+
+
+class Vote(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    chore = models.ForeignKey(Chore, on_delete=models.CASCADE)
```

Comment:
Does this line mean the vote will get deleted if the chore is deleted or vice versa?

Reply:
the vote will get deleted if chore is deleted , same regarding user